src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props

```diff
@@ -94,6 +94,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   </PropertyGroup>
 
   <PropertyGroup>
+    <DebugType Condition="'$(DebugSymbols)' == 'false'">None</DebugType>
```

Comment:
I was undecided as to what I should do if DebugType already has a value. Open to suggestions.

Reply:
I think `DebugSymbols` should take precedence. Honestly its very opinionated but the philosophy of the property / the whole point it exists is to disable the pdbs, so if it doesn't work when the other property is set it feels not so useful.

Furthermore, `DebugSymbols` has been around longer than `DebugType`. DebugSymbols is at least 8 years old.
https://github.com/MicrosoftDocs/visualstudio-docs/commit/381d67e